web/styles/compose.css

```diff
@@ -324,6 +324,7 @@
     bottom: 0;
     left: 0;
     z-index: 4;
+    transition: all 400ms;
```

Comment:
This makes me a little nervous, because if we did do the right thing and move to state classes around the compose box, this line could cause a bunch of issues. At the very least, this should be very specific about which properties (is it just z-index?) are to be transitioned, rather than `all`

Reply:
I have changed it from 'all' to 'z-index'.